pkgs/applications/science/chemistry/marvin/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, fetchurl, dpkg, jre, makeWrapper }:
+
+let
+  pname = "marvin";
+  version = "19.1.0";
+  src =
+    fetchurl {
+      name = "marvin-${version}.deb";
+      url = "http://dl.chemaxon.com/marvin/${version}/marvin_linux_19.1.deb";
+      sha256 = "1ccsimfvms5q4prjyk6sg5hsc3hkcjjfq3gl7jjm8dgd2173zzyc";
+    };
+in stdenv.mkDerivation {
+  inherit version;
+  inherit pname;
+  inherit src;
+  buildInputs = [ dpkg makeWrapper ];
+  unpackPhase = ''
+    mkdir pkg/opt -p
+    dpkg-deb -x $src pkg/opt
```

Comment:
you can directly unpack to `$out/opt` if you wish so.

Reply:
Thanks, done in 1e14e08.